test/helpers/setup-svelte-compiler.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const addHook = require('pirates').addHook;
+const { compile } = require('svelte');
+
+const SVELTE_EXTENSION = '.html';
+
+// We use LESS which requires async preprocessing, which we can't easily do here.
+// To circumvent preprocessing, we simply remove all style tags from the raw code.
+const removeStyles = code => code.replace(/<style([^]*?)>([^]*?)<\/style>/i, '');
```

Comment:
Nice comment! If you copied the regex from somewhere, it would be cool to have a link in here too. It's fine as is and not too complex anyway.